tests/bluetooth/mesh_shell/prj.conf

```diff
@@ -55,6 +55,7 @@ CONFIG_BT_MESH_TX_SEG_MSG_COUNT=2
 CONFIG_BT_MESH_RX_SEG_MSG_COUNT=2
 CONFIG_BT_MESH_RX_SEG_MAX=32
 CONFIG_BT_MESH_TX_SEG_MAX=32
+CONFIG_BT_MESH_LARGE_COMP_DATA_SRV=y
```

Comment:
I do not think that the Large Composition data server should be enabled by default due to the Health server metadata. I see all Mesh1.1 models are disabled by default.